src/content/docs/quotas.mdx

```diff
@@ -29,3 +29,55 @@ The Running Sandboxes Quota is the maximum number of Sandboxes that can be in Ru
 The Images Quota is the maximum number of Images that can be created in an Organization.
 
 The Total image size Quota is the maximum amount of disk space that can be used by Images in an Organization expressed in gigabytes.
+
+## Default Quota Allocations for New Organizations
+
+New Organizations are provisioned with the following default quotas:
+
+- **Sandboxes:** 10 total  
+- **Running Sandboxes:** 10 concurrently running  
+- **Images:** Up to 5 images  
+- **Image Storage:** 10 GB total 
+- **vCPU:** 20 vCPUs  
+- **Memory:** 40 GB RAM
+- **Disk:** 50 GB persistent storage
+
+These values are subject to change and may vary depending on your plan. To request an increase, contact [support@daytona.io](mailto:support@daytona.io).
+
+## Quota Enforcement Behavior
+
+When an Organization reaches 100% usage of any quota:
+
+- **New sandboxes, images, or other resources cannot be created** until usage falls below the limit or a quota increase is granted.  
+- **Existing sandboxes are granted a 24-hour grace period**, allowing users time to save their work and shut down environments cleanly.
+
+This approach maintains system stability while providing a buffer for active sessions.
+
+## Billing vs. Quotas
+
+Quotas define the maximum resources your Organization can allocate.
+
+**Billing is based on actual usage, not quota limits.** You are only billed when sandboxes are running and actively consuming CPU, memory, or storage.
+
+Inactive or stopped sandboxes do not incur compute costs.
```

Comment:
Stopped sandboxes still get charged for disk usage. Only archived (or destroyed) sandboxes do not have any cost.